src/commands/config/set.ts

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { Flags } from '@oclif/core';
+import { Config, Messages, Org, SfdxError, OrgConfigProperties } from '@salesforce/core';
+import { CONFIG_HELP_SECTION, ConfigCommand, ConfigResponses } from '../../config';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-settings', 'set');
+
+export class Set extends ConfigCommand<ConfigResponses> {
+  public static readonly description = messages.getMessage('description');
+  public static readonly summary = messages.getMessage('summary');
+  public static readonly examples = messages.getMessages('examples');
+
+  public static readonly strict = false;
+
+  public static readonly flags = {
+    global: Flags.boolean({
+      char: 'g',
+      summary: messages.getMessage('flags.global.summary'),
+    }),
+  };
+
+  public static configurationVariablesSection = CONFIG_HELP_SECTION;
+
+  public async run(): Promise<ConfigResponses> {
+    const { flags } = await this.parse(Set);
+    const config: Config = await this.loadConfig(flags.global);
+    let value = '';
+    const configs = await this.parseConfigKeysAndValues();
+    for (const name of Object.keys(configs)) {
+      try {
+        value = configs[name];
```

Comment:
avoids `let`, probably safe outside try/catch
```suggestion
    const configs = await this.parseConfigKeysAndValues();
    for (const name of Object.keys(configs)) {
      const value = configs[name] ?? '';
      try {
```